package.json

```diff
@@ -9,10 +9,9 @@
     "nsqlookupd"
   ],
   "dependencies": {
-    "batch": "~0.5.0",
-    "debug": "^2.2.0",
-    "superagent": "~3.5.0",
-    "superagent-retry": "^0.5.1"
```

Comment:
No longer maintained and the retry function has been integrated in superagent: https://github.com/segmentio/superagent-retry/tree/master#%EF%B8%8F-unmaintained-%EF%B8%8F